### ISSUE TITLE
The customer testimonials are not translated to English when switching

### ISSUE THREAD

**Issue by nickdotmulder** [HypoFriend/HypoFriend#16]
For example, the following is still in German despite the site being set to English.

<img width="992" alt="screenshot 2017-05-20 12 58 51" src="https://cloud.githubusercontent.com/assets/2698549/26275220/312401f4-3d5c-11e7-9e39-0cd590c3f08a.png">

**Comment by bstarr322** [comment]
Pls provide me English texts since I don't speak German.

**Comment by nickdotmulder** [comment]
@hypostef please provide copy.

**Comment by hypostef** [comment]
The translations are as follows:

"With the help of HypoFriend I was able to refinance my current mortgage and profit from today's low interest rates."

Thomas, 37, Berlin



"I had always thought it would be too much effort to refinance my old mortgage to a new bank. However, my mortgage advisor from HypoFriend identified a cheaper mortgage lender and did all the bureaucratic work of refinancing."

Saskia, 35, Hamburg


"For years I have been paying too high of an interest rate for my mortgage. With HypoFriend's expert advice I was able to save thousands of euros with minimal effort and no additional cost."

Alex, 40, Mainz


"By refinancing my mortgage I am now paying lower interest costs and can repay my mortgage faster. This means I will be debt-free much earlier than expected."

Gabriele, 44, Stuttgart
